api/receiver/create_run_test.go

```diff
@@ -45,27 +45,28 @@ func TestHandleResultsCreate(t *testing.T) {
 	assert.Nil(t, err)
 	req := httptest.NewRequest("POST", "/api/results/create", strings.NewReader(string(body)))
 	req.SetBasicAuth("_processor", "secret-token")
-	testRunIn := &shared.TestRun{
-		ID:        12345,
-		TimeStart: time.Date(2018, time.June, 21, 18, 39, 54, 218000000, time.UTC),
-		TimeEnd:   time.Date(2018, time.June, 21, 20, 3, 49, 0, time.UTC),
-		Labels:    []string{"foo", "bar"},
-		ProductAtRevision: shared.ProductAtRevision{
-			Product: shared.Product{
-				BrowserName:    "firefox",
-				BrowserVersion: "59.0",
-				OSName:         "linux",
-				OSVersion:      "4.4",
-			},
-			Revision:         sha[:10],
-			FullRevisionHash: sha,
+	pAtR := shared.ProductAtRevision{
```

Comment:
`pAtR`... I do not love this name. I do not have a better suggestion that is still 'golang short', but... damn, quite the jumble of letters :P

Reply:
To be honest, me neither, just trying to drink the Go kool aid here.